src/style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
+  width: 300px;
+  height: 100px;
+  margin: 0 auto;
+}
+
+#header {
+  background-color: blue;
+}
+
+#header:hover {
```

Comment:
you can combine similar styles like this 
`#header:hover, #footer:hover {
  ...
}`